community-content/pytorch_text_classification_using_vertex_sdk_and_gcloud/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb

```diff
@@ -1834,7 +1834,7 @@
         "%%writefile ./custom_container/Dockerfile\n",
```

Comment:
<p>Current naming for services is "Vertex AI [service name]" - please replace in Vertex Predictions, Vertex Endpoints, etc. throughout notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='214'/>

Reply:
Fixed as per current naming